src/widgets/mappainter.cpp

```diff
@@ -0,0 +1,266 @@
+#include "mappainter.hpp"
+#include <QPainter>
+#include <QPropertyAnimation>
+
+/* Constuctor */
+MapPainter::MapPainter(QWidget* parent)
+    : QWidget(parent)
+{
+    resize(parent->size());
+
+    /* Make an instance of an AirplanePainter with MapPainter as it's parent and set it's size */
+    m_airplane = new AirplanePainter(this);
+    m_airplane->resize(26,26);
```

Comment:
Magic number, why?

Reply:
Going to make a public static const for plane size